dbt/adapters/bigquery/connections.py

```diff
@@ -449,19 +450,22 @@ def execute(
             num_rows = query_table.num_rows
             num_rows_formated = self.format_rows_number(num_rows)
             bytes_processed = query_job.total_bytes_processed
+            slot_millis = query_job.slot_millis
```

Comment:
My instinct would be to name this `slot_ms`, but I'm happy to go with BigQuery's chosen name.

Also: This code is reaching a point of duplication, between different query types, where it feels prime for a refactor. Out of scope for this PR.

Reply:
Good point. I was wondering if we use `slot_ms` or the original one of the python SDK. If `bytes_processed` derives from more BigQuery's chosen name rather than the python SDK, we can go with `slot_ms`. I will modify the code.

I would also love to create a documentation of the adaptor responses for users. I will address in another issue and pull request.